tests/user/get-my-account.test.ts

```diff
@@ -0,0 +1,37 @@
+import { getMyAccount } from "@/lib/user/actions/get-my-account";
+import { withDb } from "../_db_helpers";
+import { getUserActor, insertUser } from "../_fixtures";
+import { USER_RESIDENCY } from "@/lib/data/db-enums";
+
+describe("getMyAccount", () => {
+  it("should return the correct user details", async () => {
+    await withDb(async (dbPool) => {
+      const { userId } = await insertUser(1, dbPool);
+      const actor = getUserActor(dbPool, userId);
+      const account = await getMyAccount(actor);
+
+      const {
+        userId: user_id,
```

Comment:
Seeing this again makes me wonder if we really need user ID in MyAccount.

Since users can only operate on their own account, any interaction with pawtal pertaining to MyAccount should always rely on their logged in identity. E.g. in get-my-pets, the userId comes from the UserActor.

If we remove it then we won't have this `userId: user_id` annoyance. 🙂

Reply:
Done! b07f27143d693cce888d59d16c08072d117813f4

Yea I think you're right. In that case should I modify the `get-my-latest-call.ts` as well?

Reply:
Similarly removed it for the latest call retrieval. 81ffaccbe3bad488d3840f9fffe2cec8a1b0871b